backend/handlers/handlers.go

```diff
@@ -1438,6 +1439,32 @@ func (handlers *Handlers) getExchangeMoonpayBuyInfo(r *http.Request) (interface{
 	return resp, nil
 }
 
+func (handlers *Handlers) getExchangeBtcDirectInfo(r *http.Request) interface{} {
+	type result struct {
+		Success bool    `json:"success"`
+		Url     string  `json:"url"`
+		ApiKey  string  `json:"apiKey"`
+		Address *string `json:"address"`
+	}
+
+	code := accountsTypes.Code(mux.Vars(r)["code"])
+	acct, err := handlers.backend.GetAccountFromCode(code)
+	accountValid := acct != nil && acct.Offline() == nil && !acct.FatalError()
+	if err != nil || !accountValid {
+		return result{Success: false}
+	}
+
+	action := exchanges.ExchangeAction(mux.Vars(r)["action"])
+	btcInfo := exchanges.BtcDirectInfo(action, acct, handlers.backend.DevServers())
```

Comment:
I believe here it should pass Testing() to get the prod url with `make servewallet-mainnet` 

```suggestion
	btcInfo := exchanges.BtcDirectInfo(action, acct, handlers.backend.Testing())
```

Reply:
@thisconnect I think this is correct. `DevServers ()` is true for both `make servewallet` and `make servewallet-mainnet`, while it is false for `servewallet-mainnet-prodservers`

Reply:
ack, I forgot about `make servewallet-mainnet-prodservers` makes sense.

Reply:
in this case, lets get this merged? @Beerosagos ?

Reply:
Left a few comments